.gitignore

```diff
@@ -1,2 +1,3 @@
 build
-requirements.txt
\ No newline at end of file
+lib
+requirements.txt
```

Comment:
why do we have requirements.txt here btw?

Reply:
Because I was copying the one from core and I thought it made sense but it doesn't, removing it, good catch.